src/content/en/2025/cookies.md

```diff
@@ -3,18 +3,646 @@
+      </tr>
+      <tr>
+        <td>p75</td>
+        <td class="numeric">12</td>
+        <td class="numeric">15</td>
+        <td class="numeric">22</td>
+      </tr>
+      <tr>
+        <td>p90</td>
+        <td class="numeric">21</td>
+        <td class="numeric">39</td>
+        <td class="numeric">43</td>
+      </tr>
+      <tr>
+        <td>p99</td>
+        <td class="numeric">45</td>
+        <td class="numeric">400</td>
+        <td class="numeric">396</td>
+      </tr>
+      <tr>
+        <td>max</td>
+        <td class="numeric">178</td>
+        <td class="numeric">801</td>
+        <td class="numeric">831</td>
+      </tr>
+    </tbody>
+  </table>
+  <figcaption>{{ figure_link(caption="Statistics for number of cookies set on mobile pages.") }}</figcaption>
+</figure>
+
+Websites set a median of 9 cookies of any type overall, 7 first-party cookies, and 5 or 6 third-party cookies. The tables above report several other statistics about the number of cookies observed per website and the figures below display their cumulative distribution functions (cdf). For example: on desktop a maximum of 178 first-party and 885 third-party cookies are set per website.
+
+{# TODO: if any other idea of what to say here, feel free to add #}
+
+{{ figure_markup(
+  image="number-cookies-cdf-desktop.png",
+  caption="Number of cookies per website (cdf) for desktop pages.",
+  description="The graph shows the cumulative distribution function for the number of cookies set on desktop pages. We see that more websites have a number of first-party cookies that is closer to the maximum of first-party cookies observed, than for third-party cookies.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSzdHAn-vwJ-Z05NYWZrImgKaX0q5D-jgWay8FD9lMDj2jr3cEjozE083JOSi6cZZX37vVD2TjEKw28/pubchart?oid=160162622&format=interactive",
+  sheets_gid="1535389309",
+  sql_file="nb_cookies_cdf.sql"
+  )
+}}
+
+{{ figure_markup(
+  image="number-cookies-cdf-mobile.png",
+  caption="Number of cookies per website (cdf) for mobile pages.",
+  description="The graph shows the cumulative distribution function for the number of cookies set on mobile pages. We see that more websites have a number of first-party cookies that is closer to the maximum of first-party cookies observed, than for third-party cookies. Additionally, we observe very similar results for both desktop and mobile websites.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSzdHAn-vwJ-Z05NYWZrImgKaX0q5D-jgWay8FD9lMDj2jr3cEjozE083JOSi6cZZX37vVD2TjEKw28/pubchart?oid=569578852&format=interactive",
+  sheets_gid="1448286433",
+  sql_file="nb_cookies_cdf.sql"
+  )
+}}
+
+## Size of cookies
+
+<figure>
+  <table>
+    <thead>
+      <tr>
+        <th>Size of cookies (desktop top 1M) in bytes </th>
+        <th>First-party</th>
+        <th>Third-party</th>
+        <th>All</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>min</td>
+        <td class="numeric">1</td>
```

Comment:
1. Unclear what this means/how it is calculated. Is this the average of all cookies observed? If yes, duplicates are ignored? Or the average sum of cookie data per website? 
2. Also what about value + name or only value? 
3. Max might also be restricted by measurement methodology. I would expect at least some sites to set something larger but that is rejected or truncated by the browser (maybe with a warning/issue tab note).